src/test/resources/google/cloud/compute/v1/compute.proto.baseline

```diff
@@ -23,195 +23,468 @@ option ruby_package = "Google::Cloud::Compute::V1";
 //
 // Messages
 //
+// Use global external addresses for GFE-based external HTTP(S) load balancers in Premium Tier.
+//
+// Use global internal addresses for reserved peering network range.
+//
+// Use regional external addresses for the following resources:
+//
+// - External IP addresses for VM instances - Regional external forwarding rules - Cloud NAT external IP addresses - GFE based LBs in Standard Tier - Network LBs in Premium or Standard Tier - Cloud VPN gateways (both Classic and HA)
+//
+// Use regional internal IP addresses for subnet IP ranges (primary and secondary). This includes:
+//
+// - Internal IP addresses for VM instances - Alias IP ranges of VM instances (/32 only) - Regional internal forwarding rules - Internal TCP/UDP load balancer addresses - Internal HTTP(S) load balancer addresses - Cloud DNS inbound forwarding IP addresses
+//
+// For more information, read reserved IP address.
+//
+// (== resource_for {$api_version}.addresses ==) (== resource_for {$api_version}.globalAddresses ==)
```

Comment:
Any feelings about removing this type of mark-up? It seems internal-only. And it would probably look weird in language-specific doc systems. Is there any reason not to either exclude it, or reformat it to be more human friendly? (Note that we may also be adding to GAPIC method comments the manifest metadata and possibly some default values.)

Reply:
At this point converter does not process the comments in any way, except the new lines. Doing anything with the comments is possible, but it can be done later as a logical improvement step if proven to be worth it. To not bloat the scope of this PR I would like to keep it as transferring documentation to the proto file without any processing of it.

As for this particular line, it feels like it should have been removed on the step of creating the discovery doc itself...

Reply:
OK to processing this later. Filed #5